Source/WebCore/PAL/pal/spi/cocoa/WritingToolsSPI.h

```diff
@@ -41,12 +41,14 @@ constexpr auto PlatformWritingToolsBehaviorDefault = NSWritingToolsBehaviorDefau
 constexpr auto PlatformWritingToolsBehaviorLimited = NSWritingToolsBehaviorLimited;
 constexpr auto PlatformWritingToolsBehaviorComplete = NSWritingToolsBehaviorComplete;
 
-using PlatformWritingToolsAllowedInputOptions = NSWritingToolsAllowedInputOptions;
+// FIXME: (rdar://130540028) Remove uses of the old WritingToolsAllowedInputOptions API in favor of the new WritingToolsResultOptions API, and remove staging.
 
-constexpr auto PlatformWritingToolsAllowedInputOptionsPlainText = NSWritingToolsAllowedInputOptionsPlainText;
-constexpr auto PlatformWritingToolsAllowedInputOptionsRichText = NSWritingToolsAllowedInputOptionsRichText;
-constexpr auto PlatformWritingToolsAllowedInputOptionsList = NSWritingToolsAllowedInputOptionsList;
-constexpr auto PlatformWritingToolsAllowedInputOptionsTable = NSWritingToolsAllowedInputOptionsTable;
+using PlatformWritingToolsResultOptions = NSUInteger;
+
+constexpr auto PlatformWritingToolsResultPlainText = (PlatformWritingToolsResultOptions)(1 << 0);
+constexpr auto PlatformWritingToolsResultRichText = (PlatformWritingToolsResultOptions)(1 << 1);
+constexpr auto PlatformWritingToolsResultList = (PlatformWritingToolsResultOptions)(1 << 2);
+constexpr auto PlatformWritingToolsResultTable = (PlatformWritingToolsResultOptions)(1 << 3);
```

Comment:
It looks like we can move this outside of the platform-specific paths? Since it's the same declaration in both cases.

Reply:
I'd rather not, since once I can remove the old symbols, all of this will just go back to being platform-dependent anyways, so less code will required to be changed.

Reply:
Oh okay, I didn't realize that. Np!